src/python/WMCore/WebTools/RESTApi.py

```diff
@@ -107,25 +107,25 @@ def _buildResponse(self, args=[], kwargs={}):
         except HTTPError as h:
             # If something raises an HTTPError assume it's something that should
             # go to the client
-            response.status = h[0]
-            for kwarg in kwargs.keys():
+            response.status = h.args[0]
```

Comment:
should we make sure `args[X]` is used **everywhere** we have an HTTPError exception in WMCore? Is it all included in this PR?

Reply:
Yeah, we should make sure that this is the case or this is will fail in py3! 

This is part of the `pylint --py3k` report though and it is `W1624`, so I would continue to address this one slice at a time instead of having one PR just for this type of change. I can run pylint on all the codebase to find all of the occurences though if you prefer!

Reply:
No worries Dario, let's stick to the plan and we deal with those when we find them in the respective slices. Thanks